### PR TITLE
Personal Training sync between devs

### PR DESCRIPTION
**Pull request by retif**
**Related Issue/Ticket:**

https://openy.atlassian.net/browse/PRODDEV-192

## Steps to test:

- [ ] Enable openy_gc_personal_training module
- [ ] Create personal training in admin panel
- [ ] Open personal training page

## Quality checks:

Please check these boxes to confirm this PR covers the following cases:

- Maintaining our upgrade path is essential. Check one or the other:
  - [ ] This PR  provides updates via `hook_update_N` or other means.
  - [ ] No updates are necessary for this change.
- Front end fixes should be tested against all of the Open Y Themes.
  - [ ] Tested against Carnation
  - [ ] Tested against Lily
  - [ ] Tested against Rose
  - [ ] This change does not contain front-end fixes.
- [ ] I have flagged this PR "Needs Review" or pinged the VY devs/QA
  team in Slack

### CODE REVIEW

**Review by anpolimus — APPROVED** [review]

**Comment by hamrant** [comment]
retest

File: js/gated-content/src/views/PersonalTrainingPage.vue [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+          this.video.attributes.equipment = [];
+          if (response.data.included.length > 0) {
+            response.data.included.forEach((ref) => {
+              if (ref.type === 'taxonomy_term--gc_equipment') {
+                this.video.attributes.equipment.push(ref.attributes);
+              }
+            });
+          }
+          this.loading = false;
+          console.log(this.video);
```
Comment (hamrant) [inline js/gated-content/src/views/PersonalTrainingPage.vue:146]: Please remove `console.log`

**Review by hamrant — CHANGES_REQUESTED** [review]

File: js/gated-content/src/views/PersonalTrainingPage.vue [hunk 2]
```diff
@@ -16,7 +16,6 @@
               :bundle="'virtual_meeting'"
               class="rounded-border border-concrete"
             ></AddToFavorite>
-            <AddToCalendar :event="event"></AddToCalendar>
```
Comment (hamrant) [inline js/gated-content/src/views/PersonalTrainingPage.vue]: @NightHunterSV please note, `AddToCalendar` was removed

**Review by hamrant — APPROVED** [review]

**Comment by retif** [comment]
retest

**Pull request merged**